application/src/main/resources/application.yml

```diff
@@ -34,9 +34,14 @@ spring:
     name: ehrbase
 
   cache:
+    type: jcache # change to type redis if usage of redis distributed cache is intended
     jcache:
       config: classpath:ehcache.xml
 
+  redis:
```

Comment:
please add a comment that those are only used if type redis

Reply:
added comments as requested